### DISCUSSION TITLE
Cannot create multiple DocumentStore objects in Embedded mode

### DISCUSSION THREAD

**Discussion by FactPoint** [ravendb/ravendb#19309] [Q&A]
Hi,

I'm having trouble accessing a RavenDB database from multiple processes in embedded mode. The first console app connects successfully, but subsequent apps throw errors. This issue doesn't occur in server mode. What's the recommended approach for sharing a RavenDB database across multiple processes in embedded mode?"

The error is:

System.InvalidOperationException: Cannot open database because RavenDB was unable create file lock on: 'C:\RavenData\System\system.lock'. File system type: NTFS
 ---> System.IO.IOException: The process cannot access the file 'C:\RavenData\System\system.lock' because it is being used by another process.

**Comment by ayende** [comment]
In embedded mode, we are spawning a separate process to run the RavenDB database.
If two applications try to run on the same database, that would fail. However, you can connect to the _other_ database as normal (not using embedded)
